tests/test_case.py

```diff
@@ -6,11 +6,11 @@
 from faunadb.errors import NotFound
 from faunadb.objects import Ref
 
-_FAUNA_ROOT_KEY = environ["FAUNA_ROOT_KEY"]
+_FAUNA_ROOT_KEY = environ.get("FAUNA_ROOT_KEY", "secret")
```

Comment:
Actually thinking about this more, setting all these like this forces any end users who want to run tests unable to do so without overriding all the stuff we do here. Can we just revert these changes and configure the travis env to set these?